spec/api/v1/trade_leads_spec.rb

```diff
@@ -25,7 +26,7 @@
 
       it 'returns trade leads sorted by published_date:desc, country: asc' do
         json_response = JSON.parse(response.body)
-        expect(json_response['total']).to eq(13)
+        expect(json_response['total']).to eq(16)
```

Comment:
Maybe later on we can divide the `expected_results_v1.json` file into multiple files, that way it will be less confusing why we expect the `total` to be `16` but then we match `expected_results.first(10)`, what do you think?